tests/util/sampler.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC
+
+import tensorflow as tf
+
+from trieste.type import TensorType
+from trieste.utils import DEFAULTS
+
+
+class PseudoBatchReparametrizationSampler(ABC):
```

Comment:
>Shouldn't this inherit from `trieste.acqusition.Sampler`  

hmmm, in that case we may need also passing in a not used pseudo `Probabilistic` model for `PseudoBatchReparametrizationSampler` instantiation,  which is a bit cumbersome to me, is it worth doing this?

> It may be worth moving this to the same directory, as similarly it's a toy utility for experimentation rather than a something used to implement the rest of trieste.    

Am a bit hesitate to do such a move like `objectives`, please correct me if am wrong here: as the current `PseudoBatchReparametrizationSampler` is only expected to be used within tests, unless somewhere else expect this functionality as well, it is safe to stay within tests?

Reply:
Now it has been modified to inherit from the `BatchReparametrizationSampler`